src/libs/actions/IOU.js

```diff
@@ -134,6 +169,14 @@ function payIOUReport({
             // iouReport being fetched here must be open, because only an open iouReoport can be paid.
             // Therefore, we should also sync the chatReport after fetching the iouReport.
             fetchIOUReportByIDAndUpdateChatReport(reportID, chatReportID);
+
+            // Once we have successfully paid the IOU we will transfer the user to their platform of choice if they have
+            // selected something other than a manual settlement e.g. Venmo or PayPal.me
+            if (paymentMethodType === CONST.IOU.PAYMENT_TYPE.PAYPAL_ME) {
+                Linking.openURL(buildPayPalPaymentUrl(amount, submitterPayPalMeAddress, currency));
+            } else if (paymentMethodType === CONST.IOU.PAYMENT_TYPE.VENMO) {
+                Linking.openURL(buildVenmoPaymentURL(amount, submitterPhoneNumber));
+            }
```

Comment:
What I'm a little worried about here, is that deep-linking into the Venmo app on mobile will prevent `fetchIOUReportByIDAndUpdateChatReport` from completing the Onyx merge as the app is backgrounded. 

I think this is pretty unlikely tbh, but will make sure to test this.

Reply:
1. We don't know if this is true
2. I'm not sure I know how to prevent it if it is... probably make `Onyx.merge()` return a `Promise` which we might be doing soon, but isn't really the intended behavior.

So I'm gonna suggest we do nothing about this for now until it's a problem.

Reply:
Yeah, I don't think we need to do anything here.